ecl/provider_connectivity/v2/tenant_connections/testing/fixtures.go

```diff
@@ -0,0 +1,639 @@
+const CreateAttachVnaRequest = `
+{
+    "tenant_connection": {
+        "name": "attach_vna_name",
+        "description": "attach_vna_desc",
+		"tags": {
+			"test_tags1": "test1"
+		},
+		"tenant_connection_request_id": "67d76b00-3804-11ea-8088-525400060300",
+		"device_type": "ECL::VirtualNetworkAppliance::VSRX",
+		"device_interface_id": "interface_2",
+		"device_id": "c291f4c4-a680-4db0-8b88-7e579f0aaa37",
+		"attachment_opts": {
+    		"fixed_ips": [
+        		{
+            		"ip_address": "192.168.1.3"
+        		}
+    		]
+		}
+    }
+}
+`
+
+// CreateAttachVnaResponse provides the output from a Create request.
+const CreateAttachVnaResponse = `
+{
+    "tenant_connection":{
+        "id": "f6331886-3804-11ea-95a8-525400060400",
+        "tenant_connection_request_id": "67d76b00-3804-11ea-8088-525400060300",
+        "name": "attach_vna_name",
+        "description": "attach_vna_desc",
+        "tags": {
+			"test_tags1": "test1"
+		},
+        "tenant_id": "7e91b19b9baa423793ee74a8e1ff2be1",
+        "name_other": "",
+        "description_other": "",
+        "tags_other": {},
+        "tenant_id_other": "c7f3a68a73e845d4ba6a42fb80fce03f",
+        "network_id": "061dbaa9-a3e0-4343-b3fc-0a619db66854",
+		"device_interface_id": "interface_2",
+        "device_type": "ECL::VirtualNetworkAppliance::VSRX",
+        "device_id": "c291f4c4-a680-4db0-8b88-7e579f0aaa37",
+        "port_id": "",
+        "status": "active"
+    }
+}
+`
+
+// UpdateRequest provides the input to as Update request.
+const UpdateRequest = `
+{
+  "tenant_connection": {
+    "name": "update_name",
+    "description": "update_desc",
+    "tags": {
+      "update_tags": "update"
+    },
+	"name_other": "update_name_other",
+    "description_other": "update_desc_other",
+    "tags_other": {
+      "test_tags_other": "update"
+    }
+  }
+}
+`
+
+// UpdateResult provides an update result.
+const UpdateResult = `
+{
+    "tenant_connection":{
+        "id": "ea5d975c-bd31-11e7-bcac-0050569c850d",
+        "tenant_connection_request_id": "90381138-b572-11e7-9391-0050569c850d",
+        "name": "update_name",
+        "description": "update_desc",
+        "tags": {
+			"update_tags": "update"
+		},
+        "tenant_id": "c7f3a68a73e845d4ba6a42fb80fce03f",
+        "name_other": "update_name_other",
+        "description_other": "update_desc_other",
+        "tags_other": {
+			"test_tags_other": "update"
+		},
+        "tenant_id_other": "7e91b19b9baa423793ee74a8e1ff2be1",
+        "network_id": "c4d5fc41-b7e8-4f19-96f4-85299e54373c",
+        "device_type": "ECL::Compute::Server",
+        "device_id": "7cc34d4b-a345-4e51-b3d9-62540faca7bf",
+        "device_interface_id": "",
+        "port_id": "c9c3de44-0720-4acd-87c1-9c76f0f77cac",
+        "status": "down"
+    }
+}
+`
+
+// UpdateBlankRequest provides the input to as Update with blank request.
+const UpdateBlankRequest = `
+{
+  "tenant_connection": {
+    "name": "",
+    "description": "",
+    "tags": {
```

Comment:
Can you confirm actual API response!
```suggestion
    "tags": {},
```